zrml/prediction-markets/src/lib.rs

```diff
@@ -1488,17 +1492,21 @@ mod pallet {
         ) -> Result<Weight, DispatchError> {
             ensure!(market.status == MarketStatus::Proposed, Error::<T>::InvalidMarketStatus);
             let creator = &market.creator;
+            let advisory_bond_slash_amount =
+                T::AdvisoryBondSlashPercentage::get().mul_floor(T::AdvisoryBond::get());
+            let advisory_bond_unreserve_amount =
+                T::AdvisoryBond::get() - advisory_bond_slash_amount;
```

Comment:
```suggestion
                T::AdvisoryBond::get().saturating_sub(advisory_bond_slash_amount);
```

I know it can't produce an underflow, but just to be safe for future devs, who will modify the code.